javascript/packages/fury/lib/classResolver.ts

```diff
@@ -114,8 +115,26 @@ export default class SerializerResolver {
     this.registerSerializer(fury, Type.float16Array());
     this.registerSerializer(fury, Type.float32Array());
     this.registerSerializer(fury, Type.float64Array());
+
+    this.numberSerializer = this.getSerializerById(SerializerResolver.getTypeIdByInternalSerializerType(InternalSerializerType.FLOAT64));
+    this.bigintSerializer = this.getSerializerById(SerializerResolver.getTypeIdByInternalSerializerType(InternalSerializerType.INT64));
```

Comment:
Maybe not in this PR, could we rename bigint to int64? Normally bigint means an unlimited integer which can represent number of any bits